docs/tutorial/2-tilt-up.md

```diff
@@ -4,34 +4,33 @@ subtitle: Tilt Tutorial
 layout: docs
 ---
 ## Tilt Avatars
-[Tilt Avatars][repo-tilt-avatars] is a small sample project created for this guide.
+We made [Tilt Avatars][repo-tilt-avatars] so you can try Tilt without first setting it up with your own project.
 
-It consists of a Python web API backend to generate avatars and a Javascript SPA (single page app) frontend.
-If you are not a Python or Javascript guru, don't panic!
+Tilt Avatars consists of a Python web API backend to generate avatars, and a Javascript SPA (single page app) frontend.
+It doesn’t matter if you’re not a Python or Javascript guru — you don’t need to deeply understand the project code to learn about the `Tiltfile` and other Tilt concepts.
 
 ![Randomized Tilt avatar generation](/assets/img/tutorial/tilt-avatars.gif)
 
-The focus of this project is on introducing the `Tiltfile` and other Tilt concepts: the services are demonstrative to support the guide, but you do not need to understand the code within them to be successful.
-
 > **We know that no two projects are alike!**
 >
-> While, this project uses `Dockerfile`s with Docker as the build engine and `kubectl`-friendly YAML files,
-> these only cover a small subset of Tilt functionality.
-> 
+> This project uses `Dockerfile`s with Docker as the build engine and `kubectl`-friendly YAML files.
+> But that’s only a small subset of Tilt functionality.
+>
 > Even if you're using other technologies (e.g. Helm, CRDs, `podman`), we recommend starting here to learn the Tilt fundamentals.
 >
 > Once you're comfortable with how Tilt works, we've got comprehensive guides on `Tiltfile` authoring that cover these topics and much more!
 
 ## Run `tilt up` for the First Time
+Tilt brings consistency to your development not only due to ensuring a reproducible dev-environment-as-code, but launching any Tilt project is the same with the `tilt up` command, so you always have a familiar starting point.
+`tilt up` starts the Tilt control loop, which we'll explore in more detail in a moment.
```

Comment:
@hyu I realized this went straight into "run `tilt up` in your terminal now!!!!!1111" without any sort of preamble, so I wanted to give a teensy bit of context on what/why we're about to run that.

(This is the only change in this commit that didn't come from your style tweaks, so wanted to flag it)

Reply:
(Han and I talked about this offline)